roles/installer/tasks/load_bundle_cacert_secret.yml

```diff
@@ -11,4 +11,3 @@
   set_fact:
     bundle_ca_crt: '{{ bundle_cacert["resources"][0]["data"]["bundle-ca.crt"] | b64decode }}'
   no_log: "{{ no_log }}"
-  when: '"bundle-ca.crt" in bundle_cacert["resources"][0]["data"]'
```

Comment:
I don't think we should remove that condition

Deployments rely on the `bundle-ca.crt` key within the secret data so if you remove that one and user don't define it then deployment creation will fail